lib/tasks/data.rake

```diff
@@ -1,41 +1,5 @@
 DATA_MIGRATION_SERVICES = [
   # do not delete or edit this line - services added below by generator
-  'DataMigrations::BackfillCandidatesSubmissionBlocked',
-  'DataMigrations::DropBlockFraudulentSubmissionFeatureFlag',
-  'DataMigrations::BackfillMonthlyStatisticsData',
-  'DataMigrations::FixCandidateAPIUpdatedAt',
-  'DataMigrations::BackfillCarriedOverApplicationsDegreesComplete',
-  'DataMigrations::BackfillWorkHistoryStatusForCurrentCycle',
-  'DataMigrations::BackfillWorkHistoryStatus',
-  'DataMigrations::BackfillNotCompletedExplanation',
-  'DataMigrations::BackfillCoursesForNextCycle',
-  'DataMigrations::BackfillRestructuredImmigrationStatus',
-  'DataMigrations::BackfillInvalidProviderRelationshipPermissions',
-  'DataMigrations::PopulateApplicationChoiceCurrentRecruitmentCycleYear',
-  'DataMigrations::PopulateApplicationChoiceProviderIds',
-  'DataMigrations::RemoveDuplicateProvider',
-  'DataMigrations::BackfillReferencesCompleted',
-  'DataMigrations::CleanseEocChasersSentData',
-  'DataMigrations::BackfillSetupInterviewsPermission',
-  'DataMigrations::FixEmptyOfferConditions',
-  'DataMigrations::RemoveSurplusReferenceSelections',
-  'DataMigrations::BackfillCandidateAPIUpdatedAt',
-  'DataMigrations::BackfillNoneHesaDisabilitiesCodes',
-  'DataMigrations::RemoveApplicationChoicesInTheIncorrectCycle',
-  'DataMigrations::DeleteUuidlessCourses',
-  'DataMigrations::RemovePreviousCyclesCoursesFromApplicationsInTheCurrentCycle',
-  'DataMigrations::BackfillRestucturedWorkHistoryBoolean',
-  'DataMigrations::BackfillValidationErrorsServiceColumn',
-  'DataMigrations::BackfillSelectedBoolean',
-  'DataMigrations::DeleteAllSiteAudits',
-  'DataMigrations::RemoveIncorrectHesaCodes',
-  'DataMigrations::SpecifyExportTypeForTADExports',
-  'DataMigrations::SpecifyExportTypeForNotificationExports',
-  'DataMigrations::FixMisspellingOfCaribbeanEthnicGroupAndSetHesaCodes',
-  'DataMigrations::TrimQualificationDegreeTypes',
-  'DataMigrations::BackfillExportType',
-  'DataMigrations::FixLatLongFlipFlops',
```

Comment:
I think this migration might not have been run. Can we check which ones have been run on prod and remove those but keep the rest?

Reply:
Updated. Should all of these be left? And do you know why some have never been executed.

Reply:
this one in particular we never finished due to issues with timeouts, I think